diff --git a/dist/compiler-vapor.cjs.js b/dist/compiler-vapor.cjs.js
index f3c53b7b9274b58eb536758ee4979fc4d8f66e67..aeba6a0eafcf9d584191053176bab00cf28035c2 100644
--- a/dist/compiler-vapor.cjs.js
+++ b/dist/compiler-vapor.cjs.js
@@ -789,6 +789,10 @@ function analyzeExpressions(expressions) {
             end: id.end
           });
         });
+        const parentOfMemberExp = parentStack[parentStack.length - 2];
+        if (parentOfMemberExp && isCallExpression(parentOfMemberExp)) {
+          return;
+        }
         registerVariable(
           memberExp,
           exp,
@@ -1017,6 +1021,9 @@ function extractMemberExpression(exp, onIdentifier) {
       return "";
   }
 }
+const isCallExpression = (node) => {
+  return node.type === 'CallExpression' || node.type === 'OptionalCallExpression';
+};
 const isMemberExpression = (node) => {
   return node.type === "MemberExpression" || node.type === "OptionalMemberExpression" || node.type === "TSNonNullExpression";
 };
